src/inverseRobot.js

```diff
@@ -24,7 +24,16 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const swapped = {};
+
+  for (const key in robot) {
+    if (swapped[robot[key]]) {
```

Comment:
Better use `hasOwnProperty` to check if property in object. because in your case will also return true if key gets found somewhere in the prototype chain, otherwise Object.hasOwnProperty (like the name already tells us), will only return true if key is available on that object directly (its "owns" property).